packages/contracts-bedrock/src/L1/L2OutputOracle.sol

```diff
@@ -60,73 +56,98 @@ contract L2OutputOracle is Initializable, ISemver {
     event OutputsDeleted(uint256 indexed prevNextOutputIndex, uint256 indexed newNextOutputIndex);
 
     /// @notice Semantic version.
-    /// @custom:semver 1.7.0
-    string public constant version = "1.7.0";
+    /// @custom:semver 1.8.0
+    string public constant version = "1.8.0";
+
+    /// @notice Constructs the L2OutputOracle contract. Initializes variables to the same values as
+    ///         in the getting-started config.
+    constructor() {
+        initialize({
```

Comment:
Definitely would prefer these values to all be 0 in the implementation to guarantee that the contract looks 100% dead